src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -922,19 +922,24 @@ public async Task<CipherMiniResponseModel> PutArchive(Guid id)
             throw new BadRequestException("Cipher was not archived. Ensure the provided ID is correct and you have permission to archive it.");
         }
 
-        return new CipherMiniResponseModel(archivedCipherOrganizationDetails.First(), _globalSettings, archivedCipherOrganizationDetails.First().OrganizationUseTotp);
+        return new CipherResponseModel(archivedCipherOrganizationDetails.First(),
+            await _userService.GetUserByIdAsync(userId),
```

Comment:
🎨  We should use `GetUserByPrincipalAsync` to avoid an extra DB request. (see the other controller endpoints for examples)